### ISSUE TITLE
Database URL validation does not accept underscore (_)

### ISSUE THREAD

**Issue by bartekn** [stellar/bridge-server#66]

**Comment by dulanov** [comment]
@bartekn `after port` label means that this repo will be moved to `go` monorepo in near future? I should wait or it's ok try to implement it?

**Comment by bartekn** [comment]
@dulanov I think we will move first chunk of bridge server code to monorepo next week so let's wait until it happens.

**Comment by dulanov** [comment]
That's great, I exactly need time to set-up dev env and play around a little bit, furthermore this issue looks quite simple to start from.

**Comment by dulanov** [comment]
@bartekn do you have some broken database URL examples? `bridge_example.cfg` & `compiance_example.cfg` have exactly database URL with underscore:

```toml
[database]
type = "mysql"
url = "root:@/gateway_test?parseTime=true"
```

I reproduced it in Go Playground (example is based on two similar peace of code from [bridge/config/config.go](https://github.com/stellar/bridge-server/blob/master/src/github.com/stellar/gateway/bridge/config/config.go#L99), [compliance/config/config.go](https://github.com/stellar/bridge-server/blob/master/src/github.com/stellar/gateway/compliance/config/config.go#L76)) and it works without problems - https://play.golang.org/p/s-FVPnJalU.

**Comment by bartekn** [comment]
My bad, I could have explained it better in the issue description. The issue is with username, it doesn't accept usernames with `_` but it's a valid name in DB engines.

**Comment by dulanov** [comment]
Interesting, but it also works - `postgresql://other_123:pass_word@localhost:1234/otherdb_123?connect_timeout=10&application_name=myapp` (https://play.golang.org/p/wpoJb-smH6). May be some real database URL example to reproduce?

**Comment by bartekn** [comment]
```
ro_ot:pass@/gateway_test?parseTime=true
```

**Comment by dulanov** [comment]
Thank you, so the problem is with schema name which contain underscore (`_`).

**Comment by dulanov** [comment]
@bartekn but it's illegal requirement, because:

> A valid [RFC 3986](http://tools.ietf.org/html/rfc3986#page-17) URL scheme must consist of "a letter and followed by any combination of letters, digits, > plus ("+"), period ("."), or hyphen ("-")."    scheme      = ALPHA *( ALPHA / DIGIT / "+" / "-" / "." ).

**Comment by bartekn** [comment]
Yes, but it works without `_` and some users are using this like this. So we should either ask users to always include a schema prefix (`mysql://`) or append it if there is no prefix.

**Comment by dulanov** [comment]
Now I understand you point. We should add support to not fully specified database URLs, especially for cases than user don't want to specify database schema explicitly, because it's `postgres://` in case of Horizon by default.

**Comment by maria1118** [comment]
Hi this was the case with telegram domotipbot. My name Empowered_general was not being recognized. Perhaps asking Domo Dev who is brilliant and funny will get this issue fixed
